go.mod

```diff
@@ -10,9 +10,12 @@ require (
 	github.com/algorand/go-codec/codec v1.1.7
 	github.com/algorand/oapi-codegen v1.3.5-algorand5
 	github.com/getkin/kin-openapi v0.22.0
+	github.com/jackc/pgconn v1.10.0 // indirect
+	github.com/jackc/pgerrcode v0.0.0-20201024163028-a0d42d470451 // indirect
+	github.com/jackc/pgx/v4 v4.13.0 // indirect
 	github.com/labstack/echo-contrib v0.11.0
 	github.com/labstack/echo/v4 v4.3.0
-	github.com/lib/pq v1.5.1
+	github.com/lib/pq v1.10.2
```

Comment:
I'm surprised this didn't just go away. `go mod tidy` ?

Reply:
done